x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/persistence_run_key_and_startup_broad.json

```diff
@@ -49,5 +49,5 @@
   "timeline_title": "Comprehensive Registry Timeline",
   "timestamp_override": "event.ingested",
   "type": "eql",
-  "version": 6
+  "version": 8
```

Comment:
This is ok - from the manually reconciled version bumps elastic/detection-rules/pull/1950